MAUI/Parallax-View/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+title: About .NET MAUI Parallax View control | Syncfusion
+description: Learn here all about introduction of Syncfusion .NET MAUI Parallax View (SfParallaxView) control, its elements and more.
+platform: .NET MAUI
+control: SfParallaxView
+documentation: ug
+---
+
+# .NET MAUI Parallax View (SfParallaxView) Overview
+
+[SfParallaxView]() for .NET MAUI provides a perfect way for scrolling any control that implements the IParallaxView interface with a background element that translates slower than the foreground element. The [SfParallaxView]() control binds the scroll position of a foreground element (e.g., list) to a background element and moves the background element at different speeds.
```

Comment:
The [SfParallaxView]() for .NET MAUI provides a perfect way for scrolling any control that implements the IParallaxView interface with a background element that translates slower than the foreground element. The [SfParallaxView]() control binds the scroll position of a foreground element (for example, list) to a background element and moves the background element at different speeds.